.github/workflows/release.yml

```
@@ -82,7 +82,7 @@ env:
 jobs:
   setup:
     timeout-minutes: 20
-    if: github.event.pull_request.merged == true
+    if: github.event.pull_request.merged || ${{ inputs.dry-run-release }} || github.event_name == 'push'
```

ok but remember to add the condition on top about the trigger of this workflow
```suggestion
    push:
       branches:
           - develop
```

---

moreover, you removed the "== true", leave it